src/app/routes/stake/stake-list/stake-list-card/stake-list-card.component.html

```diff
@@ -0,0 +1,26 @@
+<mat-card>
+    <mat-card-header class="card-header">
+        <div>
+            <mat-card-title class="color-white-flat">{{organization}}</mat-card-title>
+            <mat-card-subtitle class="text-subtitle">Provider Since {{providerDate}}</mat-card-subtitle>
+        </div>
+        <img src="/assets/img/{{organizationImage}}" ngs class="card-header-image img-fluid" alt="card header" />
+    </mat-card-header>
+
+    <mat-card-content>
+        <p class="color-white-flat">
+          {{stakeAmount}} <span class="text-subtitle">EWT</span>
+        </p>
+        <span class="text-subtitle patrons">Accepting Patrons <i class="fas fa-check"></i></span>
+
+        <div class="card-progress {{getStakeRatingClass()}}">
+            <span>{{getStakeRatingText()}}</span>
+            <mat-progress-bar color="0FB544" mode="determinate" value="{{stakeRating}}"></mat-progress-bar>
```

Comment:
When you pass a class property it's better to use `[value]="stakeRating"`. Interpolation is mostly used for displaying text, not to pass data through @Inputs.

Reply:
Okay, for this I will use ngClass as you suggested above.